test/main.spec.ts

```diff
@@ -40,6 +40,29 @@ describe('postcodeValidator', () => {
         });
     });
 
+    test('should throw error for invalid country codes', () => {
+        expect.assertions(1);
+        expect(() => postcodeValidator('SW1A 0AA', 'GB')).toThrow('Invalid country code: GB');
+    });
+});
+
+describe('postcodeValidatorExists', () => {
+    test('should return true for existing validators', () => {
```

Comment:
Can you please also add a negative scenario for `postcodeValidatorExists` ?

Reply:
Done